<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Evolutionary causes and clinical implications of genetic diversity</title>
    <meta charset="utf-8" />
    <meta name="author" content="B치rbara Domingues Bitarello" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/tile-view/tile-view.css" rel="stylesheet" />
    <script src="libs/tile-view/tile-view.js"></script>
    <script src="libs/js-cookie/js.cookie.js"></script>
    <script src="libs/peerjs/peerjs.min.js"></script>
    <script src="libs/tiny.toast/toast.min.js"></script>
    <link href="libs/xaringanExtra-broadcast/broadcast.css" rel="stylesheet" />
    <script src="libs/xaringanExtra-broadcast/broadcast.js"></script>
    <link href="libs/animate.css/animate.xaringan.css" rel="stylesheet" />
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">

class: center, middle
&lt;style&gt;
.remark-slide-number {
  position: inherit;
}

.remark-slide-number .progress-bar-container {
  position: absolute;
  bottom: 0;
  height: 4px;
  display: block;
  left: 0;
  right: 0;
}

.remark-slide-number .progress-bar {
  height: 100%;
  background-color: gray;
}
&lt;/style&gt;






# .pink[Evolutionary causes and clinical implications of genetic diversity]

## .lilac[B치rbara Domingues Bitarello]

### .lilac[Postdoctoral Scholar]

### .lilac[Perelman School of Medicine, University of Pennsylvania]

???
Good afternoon everyone, my name is B치rbara and I am a postdoctoral scholar in the genetics department of the school of medicine at Upenn. I am excited to present my research to all of you today.
---
class: middle, center

## Similar, but different 
&lt;img src="intro/tarsila.png" width="780px" /&gt;

*"Oper치rios"*, Tarsila do Amaral (1933) [Source: www.wikiart.org]

???
A quick look around on the street, or at work, can make it clear that we, as a species, carry a lot of diversity. We differ in height, some diseases run in our families, some of us have asthama and can't see well, and so on.

Part of that diversity is rooted in genetics, while another part is due to the environment. Teasing those apart and understanding where diversity comes from is the central goal of population genetics. 
---
class: middle, left

## Big questions

### .pink[1)] How has .lilac[evolution] shaped genetic diversity?
???
In a sense, the main questions that drives my research interests have this common thread.p
--

### .pink[2)] What are some of the .lilac[most diverse genetic systems] known and .lilac[how] did they evolve?

???
What evolutionary processes shape genetic diversity
What examples do we have of extremely diverse systems and what do they tell us about selective pressures?
--

### .pink[3)] What are the .lilac[consequences of genetic diversity] and their link to common diseases?
???
Are these consequences to such diversity, i.e, the presence of 'common variants' that predispose us to disease? 
--

### .pink[4)] Can we .lilac[predict complex phenotypes] using genetics in an equitable way that encompasses all of human genetic diversity?

???
And, finally, since it seems inevitable that we are headed towards personalized genomic medicine, can it be done in a fair way?
---
class: middle, left

## .pink[Introduction]

## .lilac[Research Project 1]

## .lilac[Research Project 2]

## .lilac[Research Project 3]

## .lilac[Conclusions]

???
I will start off by giving an overview of some important concepts that are used throughout this presentation, followed by three stories I want to tell of how I addressed some of these questions. Finally, I will wrap it all up and talk about some exciting venues for my future research.
---
class: center, top
# My research: an overview
&lt;img src="meat/schemas/schemas.001.png" width="970px" /&gt;

???
I use computational and statistics to approach much of these questions. I am a biologist with a PhD in evolutionary genetics, and, as such, a lot of my research involves a mix of comparative and population genomics. More recently, I have also embarked on a journey focused on biomedical applications and implications of findings from population genomics.
---
class: center, top
# My research: zooming in
&lt;img src="meat/schemas/schemas.002.png" width="1200px" /&gt;

???
Zooming in a little bit, the three research topics I will be focusing on today will cover first the topic of MHC diversity, the most genetically diverse locus in the genomes of all vertebrates and an extreme case of balancing selection, followed by a project focused on balancing selection more broadly, and finally a project focused on  the power of genetic prediction pf phenotypes. 
---
class: center, top
# A polymorphism is .lilac[variable site] in the genome .lilac[across individuals]
&lt;img src="meat/schemas/schemas.004.png" width="1000px" /&gt;


???
Every human individual carries 3.2 Billion 'letters' in their genome, and these carry the instructions for all the proteins as well as regulation of our genomes. 
For many of those positions, we are all the same. For some of those, however, we may find more than one variant, or allele, for a given locus. These variats arise by random mutations, and their fates will be governed by may evolutionary processes.  
---
class: center, top

## Genomic diversity in the .lilac[absense of selective pressures]
&lt;img src="meat/schemas/schemas.003.png" width="1000px" /&gt;


???
Many sites in the genome evolve neutrally. Meaning that, even when a mutation arises, it has no appreciable consequence for the phenotype in a meaningful way. By meaningful I mean a way that affects the individuals fitness, that is, their reproductive success. 

For instance, when a new population is founded by a small number of individuals from a larger popualtion, there is a loss of genetic diversity by genetic drift.

Another example is admixture. When two populations from the same species, or even two related species, have different allele frequencies for many loci and they admix, the result is that more sites in the offsprings genome will be heterozygote, which is a measure of genetic diversity
---
class: center, top

## .lilac[Balancing selection] maintains diversity in populations
&lt;img src="meat/schemas/schemas.005.png" width="1000px" /&gt;

???
In addition to neutral process, natural selection can also shape genetic variation in population. If we think of this image as a population with two variants one variant (orange), Typically, when a new mutation arises (purple) that is either highly advantageous or highly deleterious, it will quickly and efficiently be targeted by natural selection, and that will result in reduced diversity, meaning that one variant will remain. 

On the other hand, another type of natural selection, BALANCING SELECTION, will actually maintain both of these hypothetical variants segregating because there is an adaptive tradeoff in doing so.
---
class: center, top

## Sometimes, it may be .lilac[beneficial] to harbour .lilac[deleterious variants]

&lt;img src="meat/schemas/schemas.020.png" width="1000px" /&gt;

???
What does this look like in practice? Let's take the classical example of sickle cell anemia. In the 1950s, it was documented that the regions of the world where malaria was endemic highly overlapped with those were sickle cell disease was prevalent. 
Alison and others found out that a genotype at a specific locus in the genome was behind this. the 'normal variant' made it so that red blood cells had their usual shape, which is healthy but also very susceptible to malaria infection. A mutant variant, on the other hand, represented by S here, made it so that blood cells had the form of a sickle, which causes a severe form of anemia, but as a sice product makes it really hard to be infected by  malaria. The intermediate or heterozygote phenotype, finally, gives some protection to malarial infection while only causing mild anemia, and this genotype harbours strong signatures of balancing selection.

this is very interesting. A detrimental variant (the S allele) is selectively maintained in populations where malaria is endemic.
---
class: middle, left
## .lilac[Introduction]

## .pink[Research Project 1: Heterozygote advantage in HLA genes]

## .lilac[Research Project 2: Balancing selection in the human genome]

## .lilac[Research Project 3: Polygenic scores in different populations]

## .lilac[Conclusions]

???

Ok, so Now that we've covered what a variant or a polymorphic site in the genome is, what balancing selection is, we can jump into some research questions

[8 minutes until here]
---
class: center, middle

### .pink[1)] How has .lilac[evolution] shaped genetic diversity?


### .pink[2)] What are some of the .lilac[most diverse genetic systems] known and .lilac[how] did they evolve?
???
Let's start by focusing on the most diverse region of the vertebrate genome: the MHC
---
class: center, top
## The MHC is the .lilac[most genetically diverse] region of the vertebrate genome

&lt;img src="meat/schemas/schemas.006.png" width="950px" /&gt;
???

The major histocompatibility was first detected as the factor behind transplant rejection, hence the name 'histocompatibility'. The reason for this is that this rgion harbours many genes respnsible for the adaptive immune response, that is, the one that we lear, not the innate one. Early in life our immune systems learn to discriminate self from non-self, and the rejection of transplants from incompatible donors comes from one immune system recognizing another as an invader. In addition to modulalting our immune responses, then, the genes in this locus have negative links to disease,, including many auto-immune diseases like celiac disease and psoriasis, to infectious disease like HIV and hepatitis,  to other complex diseases like T2 diabetes.
There is a vast literature on how certain alleles in these genes confer higher resistance to certain types of infection.

---
class: center, top
## The MHC is an .lilac[extreme instance] of balancing selection

&lt;img src="meat/schemas/schemas.007.png" width="1000px" /&gt;
???
So this is clearly a very interesting locus, both from an evolutionary standpoint, as well as a more applied, health perspective. It has been known for some time that these genes represent the strongest instance of balancing selection in humans, and there is a good mechanistic basis behind it. The MHC genes code for cell surface proteins that present a antigen to the bonafide immune cells, the T cells shown here on the left. On the middle we see a schematic of the MHC molecule binding an antigen. Take note that this grove here corresponds to the part of the MHC genes that is actually very genetically diverse. In the 70s, it was suggested that individuals that are heterozygote for HLA alleles would have higher fitness, since each allele codes for one of these chains in the molecule and a heterozygote would be able to present a wider array of antigens.

This is fascinating, because on the one hand, our immune sustems are evolving to better present antigens efficiently, while pathogens are evolving to evade our immune systems. This is behind the diversity in these loci and why it is a good exaple of host-pathogen arms race or red queen hypothesis. In alice in wonderland, the red queen effect, which proposes that species must constantly adapt, evolve, and proliferate in order to survive while pitted against ever-evolving opposing species. 
---
class: center, middle

# Do .lilac[all heterozygotes] have the same fitness?

# Is there support for the .lilac[divergent allele advantage model?]

???
This makes complete sense. A question that motivated me was whether all heterozygotes are equally adaptive. 
---
class:  middle, left

## Is there support for the .lilac[divergent allele advantage model?]

#### .pink[Hypothesis:] divergent allele advantage model (heterozygote advantage is greater for genotypes with more divergent alleles) explains selective patterns in the HLA genes

#### .pink[Prediction:] increased signatures of balancing selection between pairs of HLA alleles that match different antigens
???
In other words, is there support for a model where heterozygotes composed of alleles that present a different antigen repertoire from each other more adaptive?
---
class: center, top
### To answer these questions, I looked at the human class I HLA genes

&lt;img src="meat/schemas/schemas.008.png" width="1200px" /&gt;

???
To answer this, I looked at the three class I classical genes: HLA-A, HLA-B, HLA-C. Each one of those have thousands of different alleles described in a heavily curated immunogenetics database, and that number keeps increasing. The first step consisted in gathering the data and splitting these genes into ARS - the portions in the genome where the antigen binding sites are located - and the non-ARS - the rest of the gene. The latter were used as controls.

Now, HLA alleles are related to each other genealogically, like any other DNA sequences. Some of them are more closely related then others, and they are normally described either as supertypes (groups of alleles that present smilar sets of antigens) or allelic lineages (clades of alleles). For example this tree here is a subset of the allelic lineages for HLA-B. 

The next step was to use phylogenetic methods for hypothesis testing. The main hypothesis I tested was whether balancing selection was stronger in the branches that separate lineages than within lineages themselves. 

What I observed is that 

1) the evidence for selection is stronger between lineages, supporting the divergent allele advantage model and

2) the evidence for selection is also significant within lineages, meaning that it is also important, even if less so.
---
class: middle, left

## .lilac[Introduction]

## .lilac[Research Project 1: Heterozygote advantage in HLA genes]

## .pink[Research Project 2: Balancing selection in the human genome]

## .lilac[Research Project 3: Polygenic scores in different populations]

## .lilac[Conclusions]

???
The MHC/HLA system is certainly very interesting and unique, but it led me to other questions, and that's a good way to introduce my next research project


[16 minutes until here]
---

class: center, middle


# .lilac[How prevalent] has balancing selection been throughout human evolution? 

# What are the instances .lilac[beyond the MHC] loci?

???
What were the other isnances of balancing selection in our evolution beyong the MHC and how prevalent has this evolutionary process been?
---
class:top, left

### Can we devise a statistical test specific for signatures of balancing selection?

#### .pink[Approach:] combine independent signatures into one statistic
???
For starters this was hard to quantify because, unlike positive selection, balancing selection had not received a lot of attention in the literature, and most statistical tests used to detect it were not very specific

--

### If so, how prevalent has it been throughout human evolution?

#### .pink[Approach:] perform a genome-wide scan using a test specifically tailored for balancing selection

???
So we conjectured that by first designing a specific test for abalncing selection and then applying it to real data we would get a more complete picture

--

### Are there commn trends amongst the candidate regions/genes?

#### .pink[Approach:] Explore candidate regions/genes for common functions/patterns

???
Having used this statistic to search the genome, what can we say about the candidate targets of balancing selection in humans?
---
class:left, top

### Balancing selection results in .lilac[increased proportions] of polymorphic sites and .lilac[intermediate frequency] alleles

&lt;img src="meat/schemas/schemas.009.png" width="930px" /&gt;
???
These two trees show alleles segregating under neutral evolution and under balancing selection, and the arrow goes from the past to the present. Due to drift, neutral alleles have an expected time span before they fixate or get lost, shown in gray. A locus under balancing selection, on the other hand, will actively keep two or more variants segregating in a locus for a longer time than expected under neutral evolution.|This fact results in two characteristic signatures: 1) a higher proportion of intermediate frquency alleles, which is shown by the blue and red alleles in the tree; and 2) a higher proportion of polymorphisms (controlled by the underlying mutation rate) than a typical neutrally evolving region of the genome. So my idea in this project was to combine both of these signatures into one statistics.
---
class:left, top
## A statistical test .lilac[specifically] tailored for .lilac[balancing selection?]
&lt;img src="meat/schemas/schemas.010.png" width="1100px" /&gt;

???
Once I did that, the first step was to assess if this statistically actually worked. To do that, I simulated DNA sequences evolving under no selective pressure (i.e neutrally) and sequences where a locus is under balancing selection with different strengths. Both of these sequences were simulated 1000s of times under a model of human drmographic history so that genetic drift, bottlenecks etc cna be taken into account.Step 2 then involved calculating our new statistic (NCD2) for both sets of sequences and asking if they were sufficiently different to allow discrimination into these two categories. And indeed they were. Moreovoer, they had more power to detect balancing selection and ran more efficiently in computers than prev know methods.
---

### Pervasiveness and .lilac[targets] of balancing selection in humans

&lt;img src="meat/schemas/schemas.011.png" width="1000px" /&gt;

???
So we verified that we had a powerful new test. We then applied it to real population genetics data from the 1000G project. In particular, we focused on populations from Africa and Europe. We then ran a genome-wide scan, defining windows of fixed size along the genome and calculating the statistic for each window. That results ina  distribution, awe wnd cutoffs can be defined either based on simulations (how unexpected?) or empirical (how uncommon). We used a mix of these two approaches and found that we were able to recriate most previous candidate genes in the literature, plus a number of novel, prev underscribed genes/regions. Among the most extreme outlier windows, there was a significantly higher proportion of protein-altering variants (compared to controls), a significantly higher thane xpected proportion of genes expressed preferentially in testis, an excess of immune genes (1/3 of total) as well as an excess of genes in 'defense' categories more broadly, including immunity but also skin barriers, bitter taste receptors etc

---
class: middle, left

## .lilac[Introduction]

## .lilac[Research Project 1: Heterozygote advantage in HLA genes]

## .lilac[Research Project 2: Balancing selection in the human genome]

## .pink[Research Project 3: Polygenic scores in different populations]

## .lilac[Conclusions]

???
This project made me very aware of how a given gene may be involved in many different processes and how balancing selection may play a role in explaining common diseases. This leads us to the final research story I am sharing today, which takes on a pretty different direction, focusing on biomedical data.

As hinted by this last slide, genetics is more complicated than simply 'good variants' get selected. Maladaptive variants can get fixated because of genetic drift, other variants can be good in certain situations but not others. So this is a good way to shift into the next topic of research, which revolves around disease risk prediction
---
class:center, top
##**Many** variants with **small** effect size

.pull-left[.full-width[
&lt;img src="tam_et_al_2019_2.png" width="600px" /&gt;

Tam et al. (2019) *Nat Rev Genet*
]]

.pull-right[.full-width[

 .blue[Rare, monogenic diseases/traits:]
 
 phenylketonuria, sickle-cell anemia, Duchenne muscular dystrophia

.blue[complex, common diseases/traits:]

cardiovascular  disease, T2 diabetes, cancers, **height**, BMI
]]
???
A picture that has emerged from human genetics from a biomedical standpoint is that most genomic variants (or polymorphisms) that have a statistically significant association with a disease or trait are somewhat common (appreciable frequencies) and have a realtively small 'effect size'. On the other hand, what you may have learned as monogenic diseases/traits, such as muscualr distrofia

This results in the landscape of polygenic traits to look something like we see on the right: GWAS are mostly successful at capturing variants between these two diagonal lines: common, with moderate or small effect variants, and some rare but highly penetrant mutations, that is, that have a considerable effect size.
Rare variants with small effect sizes are difficult to identify using GWAS, and common variants with large effects are unusual for common complex diseases
---
class: inverse, middle, center

# .lilac[Combined, these variants can explain a lot!]

???

So you might be thinking this is bad news in terms of predicting outcomes. So although most of these variants we detect in genome-wide association studies have very small effect, when you add them up, they explain quite a lot of the variation in a trait. How, you may ask. let's rewind a bit.

---

class: center, top
### Genome-wide association studies .pink[(GWAS)] detect .pink[genetic associations] to traits of interest

&lt;img src="meat/schemas/schemas.015.png" width="980px" /&gt;

???
Earlier I mentioned that there were genetic associations to autoimmune diseases in the MHC region. How are those associations detected?
GWAS are big scale studies. It works roughly like this: a trait of interest is chosen. Typically a binary trait such as presence or absence of disease status or a continuous trait, like height. A large cohort of individuals has their phenotype measured, and their genetic data analysed. Then, each variant is tested as shown in this plot on the right as to wehther it has an effect in the phenotype or not. In this fake example, depending on the genotype in this variant, the individual has remarkably different phenotypes. Next, each variant is represented in a plot such as this one, called a manhattan plot, where the peaks suggest the approx geneomic location of variants associated with the trait. Finally, the results from a gwas may be used in a so called polygenic score to asess the disease/phenotype status of a invidiaul outside the initial discovery cohort. 
---
class: center, top

## Polygenic risk scores combine .lilac[all variants] with an effect on the phenotype
.pull-left[.full-width[
&lt;img src="gwas_schema_pasaniuc_price.png" width="650px" /&gt;

.small[Pasaniuc &amp; Price (2017), *Nat Rev Genet*]


]]

.pull-right[.full-width[




`$$PRS=\sum_{i=1}^{m} \hat\beta_{i} G_{j,i}$$` 


`\(\hat\beta\)` : effect size (from GWAS)

`\(G\)` : Effect allele dosage

`\(j\)`: Individuals

`\(i\)`: SNPs

**independence**

**additive model**

]]


???
So you did a GWAS for height. Then, for each variant in the genome you have something like this: the genotype in that locus has an effect on height, and can be modelled as a linear relationship. So you have an effect size for one allele in that locus, which is a correlation between the phenotype and the genotype. 

Then you make some assumptions about independence of these loci and that an additive model is appropriate and you have this polygenic score, which is a weighted sum of the genotypes in an individuals. Weighted by the effect sizes.
---
class: top, center
# PRSs are appealing


.pull-left[.full-width[
## **easy**

### **promising**

## **fast**

### **minimal requirements**
]]
.pull-right[.full-width[

#### Polygenic risk of cardiovascular disease
&lt;img src="cad.png" width="400px" /&gt;

Khera et al. (2018), *Nat Genet*
]]

???
As a mentioned a few slides ago, a somehwat new approach is to combine the significant restuls from a GWAS into one single value, the polygenic score. The advantage is that althoguh each variatn has a small effect, overall they add up to substantial amounts of information.
So as you can see from this, PRS are very appealing...They are easy and simple to implement, all you need are some summary statistics from a GWAS, and no further complicated modeling.
Here is one promising example. The x axis shows the percentile of polygenic score. And the y axis shows the pravalence of CAD. IF we look at the highest bins of PRS, prevalance of CAD is 10%. These individuals have a 8-fold increase of CAD compared to the rest of the population. This makes the PRS able to find 20-fold mroe people at risk for CAD than if you were simply testing for large effect mutations, which have large effect but are rare.
There are a few other examples like this sofar for breast cancer for instance.
The main point I want to emphasize is...This is for individuals with European ancestry.

---
class: middle, center, inverse

# .lilac[What about ancestry?]

???
right. So what about ancestry?
---
class: center, top
.pull-left[.full-width[
## European **ancestry** represent almost 80% of GWAS participants...
&lt;img src="Screen Shot 2020-09-16 at 4.28.04 PM.png" width="350px" /&gt;

Sirugo, Williams &amp; Tishkoff (2019), *Cell*]]
--
.pull-left[.full-width[

&lt;img src="index_files/figure-html/unnamed-chunk-19-1.png" width="420px" /&gt;

[Data: https://worldpopulationreview.com/]

## .. and &lt;15% of the world's population
]]

???
Ok, so let's look at this pie chart from last year, with the proportion of individuals from Each of these groups of ancestry that appear in GWAS. Europeans make up almost 80%, Asians about 20% and Africans about 2%. Note that this is a big improvement since 2009. The same survey in 2009 showed that 96% of individuals were European!

Anyway, now look at this pie chart on the right, which I made from data from worldpopulationreview for 2020. Now, the colors are not a true match, but I did my best. BEar in mind that the plot on the left is about ancestry, and the one on the right is about continents, regardless of ancestry. Nevertheless, it's a reasonable approximation for this purpose. Europeans in Blue represent a disproportionate amount of GWAS participants especially if you consider their share of the world's population. Asian POPULATION represent 60% of the world and ASIAN ANCESTRY only 10% of GWAS, and African POPULATION 17 and AND AFRICAN ANCESTRY 2%. Not to mention the other ancestries which are virtually non-existent.
---
class: center, top
### How much do polygenic scores predict phenotypes in .pink[other populations?]


&lt;img src="meat/schemas/schemas.019.png" width="1000px" /&gt;


???
this is obviously not ideal and reflects many biases which I don't have time to go into to, but considering the hype i showed about PRS before, how does this affect popygenic risk score predictions? If personalized genomics are to became a part of medical care, we need be sure that we understand how these predictions work for the mostly unsampled populations of the world.

This figure from Martin et al showes average prediction accuracies for 17 traits, using GWAS summary statisticas derived from 100% European cohorts. I am not sure what american means? As you can see, relative to the European ancestrie, all the other ancestries see a much reduced prediction accuracy for all traits

---
class: middle, center


## How do these .lilac[different factors] affect prediction accuracy?


## What can we .lilac[do about it?]


???
So the questions that motivated this work were: how 
---
class: middle, left


## How do these different factors affect prediction accuracy?

#### .pink[Approach:] examine how well we can predict a highly heritable trait in different populations, based on an European ancestry discovery cohort and explore the roles of different biological/statistical factors

## What can we do about it?

#### .pink[Approach:] explore whether incorporating fine-scale ancestry information into these scores improves their performance

???
So the questions that motivated this work were:
---
class: middle, center

# .lilac[Let's look at] height
???
We chose to look at height because it is ahighly heritable trait, that is, it has a large genetic component.
---
class:left, top
## Wow well can we predict a .lilac[highly heritable] trait?

&lt;img src="meat/schemas/schemas.017.png" width="1000px" /&gt;


???
why height? It's a somehwat well-udnerstood trait. it's hgihly popygenic and easy to measure. It is highly heritable (80% based ont twin studies, about 50% based on SNP heritability)

Also, it is rather easy to measure and is measured in most biomedical studies, so that increases the amount of available data. We used GWAS results for height that were performed in more than 3000K inds from the UK Biobank. That includes exclusively individuals whose main ancestry component is European.
Then, we selected almost 30,000 indds from independent cohorts ranging from about 0-100% african ancestry. For those individuals, we had genetic and phenotypic data, that is, their height, as well as their age and biolofical sex, important covariates for height.
We then calculated polygenic scores for height for each individual, using effect sizes from the Eurpean ancestry cohort.
Importantly, unlike most studies, we consdierted ancestry as a continuous variable going from 0-100% european ancestry. So each individuals we analysed had a "proportion of Eur ancestry" inferred by local ancestry inference and averaged across the genome.

In the end we looked at about 19,000 individuals with admied ancestry, and 10,000 with eur ancestry from different cohorts.

we modell height as a function of sex age and eur ancesty and compare that to a mdoel that also includes prs.
---
class:left, top
## Wow well can we predict a .lilac[highly heritable] trait in .lilac[different populations?]
&lt;img src="meat/schemas/schemas.018.png" width="1000px" /&gt;

???
Here we have the main finding. Explain this figure.
We then proceeded to investifate, one by one, the different methodological and statistical factors that could be driving this and concluded that there is no simply fix to this loss of prediction. That is, the fix would have to happen upstream, i.e, the discovery cohorts need to be more diverse. 
---
class:left, middle


## .lilac[Introduction]

##  .lilac[Research Theme 1: Balancing selection in humans]

## .lilac[Research Theme 2: Polygenic risk prediction for individuals with non-European ancestry] 

## .pink[Conclusions &amp; Future Directions]

???
With that I will move on the final part of this talk, where I highlight some exciting venues for feature research.
---
class: left, middle


### .pink[How has balancing selection shaped diversity in the HLA loci?]



### .pink[How prevalent has balancing selection been throughout evolution and what are the common trends?]


### .pink[How useful are polygenic risk predictors for individuals with non-European ancestries?]

???
A brief recap of what I presented in those three research projects. These were the main questions.
---
class: middle, left



### .pink[1)] There is support for the .blue[divergent allele advantage model] in HLA evolution


### .pink[2)] A novel test, specifc for balancing selection, revealed many new candidates, 1/3 of which are immune-related, and many others related to defense broadly and reproduction


### .pink[3)] At present, PRS approaches have little to no utility for individuals with non-European ancestry, many factors are responsible, need to diversify genetics studies at all stages
???
And these were the answers
---
class:left, top
&lt;img src="meat/schemas/schemas.012.png" width="1000px" /&gt;
???
Going back to this, now, I hope I have convinced you that there are many fascinating biological questions waiting to be answered, and that genomics provides powerful tools to answer many of them. My research path represented by these three projects helps contextualize my two main axis of fture research:
---
class: center, top
## Research Axis I: Balancing selection as a force that shapes diversity

&lt;img src="meat/schemas/schemas.014.png" width="1000px" /&gt;

???
Talk about this
---
class: center, top
## Research Axis II: Polygenic risk prediction in admixed populations

&lt;img src="meat/schemas/schemas.013.png" width="930px" /&gt;
???
Talk about this
---
class: top, center
# Acknowledgements

.pull-left[.full-width[

**Iain Mathieson**

**Diogo Meyer**

**Aida Andr칠s**

Joshua Schmidt

Cesare de Filippo

Rodrigo dos Santos Francisco

D칠bora Brandt

]]

.pull-right[.full-width[
Neale Lab

UK Biobank

Women's Health Initiative

Jackson Heart Study

Health and Retirement Study

1000 Genomes Project
]]


&lt;img src="fapesp-logo.jpg" width="300px" /&gt;


---

class: inverse,center, middle

# .lilac[Thank you!]

# .lilac[Questions?]




    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9",
"slideNumberFormat": "<div class=\"progress-bar-container\">\n  <div class=\"progress-bar\" style=\"width: calc(%current% / %total% * 100%);\">\n  </div>\n</div>`\n"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
