<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Evolutionary causes and clinical implications of genetic diversity</title>
    <meta charset="utf-8" />
    <meta name="author" content="Bárbara Domingues Bitarello" />
    <script src="libs/header-attrs/header-attrs.js"></script>
    <link href="libs/tile-view/tile-view.css" rel="stylesheet" />
    <script src="libs/tile-view/tile-view.js"></script>
    <script src="libs/js-cookie/js.cookie.js"></script>
    <script src="libs/peerjs/peerjs.min.js"></script>
    <script src="libs/tiny.toast/toast.min.js"></script>
    <link href="libs/xaringanExtra-broadcast/broadcast.css" rel="stylesheet" />
    <script src="libs/xaringanExtra-broadcast/broadcast.js"></script>
    <link href="libs/animate.css/animate.xaringan.css" rel="stylesheet" />
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">

class: center, middle
&lt;style&gt;
.remark-slide-number {
  position: inherit;
}

.remark-slide-number .progress-bar-container {
  position: absolute;
  bottom: 0;
  height: 4px;
  display: block;
  left: 0;
  right: 0;
}

.remark-slide-number .progress-bar {
  height: 100%;
  background-color: gray;
}
&lt;/style&gt;






# .pink[Evolutionary causes and clinical implications of genetic diversity]

### .lilac[Research Talk]

### .lilac[Bárbara Domingues Bitarello, Postdoctoral Scholar]

### .lilac[Perelman School of Medicine, University of Pennsylvania]
---
class: middle, left

## .lilac[Introduction]

## .pink[Research Project 1]

## .pink[Research Project 2]

## .pink[Research Project 3]

## .pink[Conclusions &amp; Future Directions]

---
class: center, top
# My research: an overview
&lt;img src="meat/schemas/schemas.001.png" width="970px" /&gt;

---
class: center, top
# My research: zooming in
&lt;img src="meat/schemas/schemas.002.png" width="1200px" /&gt;
---
class: center, top
# Architecture of a genome
&lt;img src="meat/schemas/schemas.004.png" width="910px" /&gt;
---
class: center, top

## Processes that shape genomic diversity in the .lilac[absense of selective pressures]
&lt;img src="meat/schemas/schemas.003.png" width="1000px" /&gt;
---
class: center, top

## .lilac[Selective processes] that shape genomic diversity
&lt;img src="meat/schemas/schemas.005.png" width="1000px" /&gt;
class: center, top
---
class: middle, left
## .lilac[Introduction]

## .pink[Research Project 1]

## .lilac[Research Project 2]

## .lilac[Research Project 3]

## .lilac[Conclusions &amp; Future Directions]
---
class: center, top

## MHC molecules present antigens and unleash immune responses

&lt;img src="meat/schemas/schemas.006.png" width="950px" /&gt;

---
class: center, top
## MHC/HLA: an extreme instance of balancing selection

&lt;img src="meat/schemas/schemas.007.png" width="1000px" /&gt;
---

class: center, middle

# Is there support for the .lilac[divergent allele advantage model?]
---
class:  middle, left

## Is there support for the divergent allele advantage model?

#### .pink[Hypothesis:] divergent allele advantage model (heterozygote advantage is greater for genotypes with more divergent alleles) explains selective patterns in the HLA genes

#### .pink[Prediction:] increased signatures of balancing selection between pairs of HLA alleles that match different antigens
---
class: center, top
### To answer these questions, I looked at the human class I HLA genes

&lt;img src="meat/schemas/schemas.008.png" width="1200px" /&gt;
---
class: middle, left

## .lilac[Introduction]

## .lilac[Research Project 1]

## .pink[Research Project 2]

## .lilac[Research Project 3]

## .lilac[Conclusions &amp; Future Directions]
---

class: center, middle


# .lilac[How prevalent] has balancing selection been throughout human evolution? 

# What are the instances .lilac[beyond the MHC] loci?

---
class:top, left

### Can we devise a statistical test specific for signatures of balancing selection?

#### .pink[Approach:] combine independent signatures into one statistic

### If so, how prevalent has it been throughout human evolution?

#### .pink[Approach:] perform a genome-wide scan using a test specifically tailored for balancing selection

### Are there commn trends amongst the candidate regions/genes?

#### .pink[Approach:] Explore candidate regions/genes for common functions/patterns
---
class:left, top

## Can we devise a statistical test specific for signatures of balancing selection?

&lt;img src="meat/schemas/schemas.009.png" width="930px" /&gt;

---
class:left, top
## A method to detect signatures of balancing selection
&lt;img src="meat/schemas/schemas.010.png" width="1200px" /&gt;
---

### Pervasiveness and targets of balancing selection in humans

&lt;img src="meat/schemas/schemas.011.png" width="1000px" /&gt;
---
class: middle, left

## .lilac[Introduction]

## .lilac[Research Project 1]

## .lilac[Research Project 2]

## .pink[Research Project 3]

## .lilac[Conclusions &amp; Future Directions]
---

class: center, top
## Genome-wide association studies (GWAS)

&lt;img src="meat/schemas/schemas.015.png" width="970px" /&gt;

???
How are associated variants detected? Introducing: the GWAS. 
class:left, top

???
As hinted by this last slide, genetics is more complicated than simply 'good variants' get selected. Maladaptive variants can get fixated because of genetic drift, other variants can be good in certain situations but not others. So this is a good way to shift into the next topic of research, which revolves around disease risk prediction

---
class:center, top
##**Many** variants with **small** effect size

.pull-left[.full-width[
&lt;img src="tam_et_al_2019_2.png" width="600px" /&gt;

Tam et al. (2019) *Nat Rev Genet*
]]

.pull-right[.full-width[

 .blue[Rare, monogenic diseases/traits:]
 
 phenylketonuria, sickle-cell anemia, Duchenne muscular dystrophia

.blue[complex, common diseases/traits:]

cardiovascular  disease, T2 diabetes, cancers, **height**, BMI
]]
???
A picture that has emerged from human genetics from a biomedical standpoint is that most genomic variants (or polymorphisms) that have a statistically significant association with a disease or trait are somewhat common (appreciable frequencies) and have a realtively small 'effect size'. On the other hand, what you may have learned as monogenic diseases/traits, such as muscualr distrofia

This results in the landscape of polygenic traits to look something like we see on the right: GWAS are mostly successful at capturing variants between these two diagonal lines: common, with moderate or small effect variants, and some rare but highly penetrant mutations, that is, that have a considerable effect size.
Rare variants with small effect sizes are difficult to identify using GWAS, and common variants with large effects are unusual for common complex diseases
---
class: inverse, middle, center

# .lilac[Combined, these variants can explain a lot!]

???

So you might be thinking this is bad news in terms of predicting outcomes. So although most of these variants we detect in genome-wide association studies have very small effect, when you add them up, they explain quite a lot of the variation in a trait.

---
class: middle, center
# Polygenic traits and their predictive power


|Phenotype     | Variance_Explained| Variants|
|:-------------|------------------:|--------:|
|height        |               25.0|     3000|
|schizophrenia |                7.0|      100|
|ADHD          |                5.5|      100|
### PS: for **Europeans** ancestry only...

???

To mention a few examples, height is one of the most well-studied and most polygenic traits we know of. So far more than 3,000 independently associated variants have been uncovered through GWAS and together they explain about 25% of variation in height as show by linear regression models. For schizophrenia and ADHD, those values are between 5.5-7%. That might seem low, but actually it isn't if you think about this...
---
class: center, top

## Polygenic risk scores combine .lilac[all variants] with an effect on the phenotype
.pull-left[.full-width[
&lt;img src="gwas_schema_pasaniuc_price.png" width="650px" /&gt;

.small[Pasaniuc &amp; Price (2017), *Nat Rev Genet*]


]]

.pull-right[.full-width[




`$$PRS=\sum_{i=1}^{m} \hat\beta_{i} G_{j,i}$$` 


`\(\hat\beta\)` : effect size (from GWAS)

`\(G\)` : Effect allele dosage

`\(j\)`: Individuals

`\(i\)`: SNPs

**independence**

**additive model**

]]


???
So you did a GWAS for height. Then, for each variant in the genome you have something like this: the genotype in that locus has an effect on height, and can be modelled as a linear relationship. So you have an effect size for one allele in that locus, which is a correlation between the phenotype and the genotype. 

Then you make some assumptions about independence of these loci and that an additive model is appropriate and you have this polygenic score, which is a weighted sum of the genotypes in an individuals. Weighted by the effect sizes.
---
class: top, center
# PRSs are appealing


.pull-left[.full-width[
## **easy**

### **promising**

## **fast**

### **minimal requirements**
]]
.pull-right[.full-width[

#### Polygenic risk of cardiovascular disease
&lt;img src="cad.png" width="400px" /&gt;

Khera et al. (2018), *Nat Genet*
]]

???
So as you can see from this, PRS are very appealing...They are easy and simple to implement, all you need are some summary statistics from a GWAS, and no further complicated modeling.

Here is one promising example. The x axis shows the percentile of polygenic score. And the y axis shows the pravalence of CAD. IF we look at the highest bins of PRS, prevalance of CAD is 10%. These individuals have a 8-fold increase of CAD compared to the rest of the population. This makes the PRS able to find 20-fold mroe people at risk for CAD than if you were simply testing for large effect mutations, which have large effect but are rare.
There are a few other examples like this sofar for breast cancer for instance.
The main point I want to emphasize is...This is for individuals with European ancestry.

---
class: middle, center, inverse

# .lilac[What about ancestry?]

???
right. So what about ancestry?
---
class: center, top
.pull-left[.full-width[
## European **ancestry** represent almost 80% of GWAS participants...
&lt;img src="Screen Shot 2020-09-16 at 4.28.04 PM.png" width="350px" /&gt;

Sirugo, Williams &amp; Tishkoff (2019), *Cell*]]
--
.pull-left[.full-width[

&lt;img src="index_files/figure-html/unnamed-chunk-18-1.png" width="420px" /&gt;

[Data: https://worldpopulationreview.com/]

## .. and &lt;15% of the world's population
]]

???
Ok, so let's look at this pie chart from last year, with the proportion of individuals from Each of these groups of ancestry that appear in GWAS. Europeans make up almost 80%, Asians about 20% and Africans about 2%. Note that this is a big improvement since 2009. The same survey in 2009 showed that 96% of individuals were European!

Anyway, now look at this pie chart on the right, which I made from data from worldpopulationreview for 2020. Now, the colors are not a true match, but I did my best. BEar in mind that the plot on the left is about ancestry, and the one on the right is about continents, regardless of ancestry. Nevertheless, it's a reasonable approximation for this purpose. Europeans in Blue represent a disproportionate amount of GWAS participants especially if you consider their share of the world's population. Asian POPULATION represent 60% of the world and ASIAN ANCESTRY only 10% of GWAS, and African POPULATION 17 and AND AFRICAN ANCESTRY 2%. Not to mention the other ancestries which are virtually non-existent.
---
class: center, top
### How much do polygenic scores predict phenotypes in .pink[other populations?]


&lt;img src="meat/schemas/schemas.019.png" width="1000px" /&gt;


???
this is obviously not ideal and reflects many biases which I don't have time to go into to, but considering the hype i showed about PRS before, how does this affect popygenic risk score predictions?
Over the past two years or so a few papers have come out showing that prediction accuracy from PRS decreases with genetic distance from Europeans.
This figure from Martin et al showes average prediction accuracies for 17 traits, using GWAS summary statisticas derived from 100% European cohorts. I am not sure what american means?

---
class: middle, center


## How do these .lilac[different factors] affect prediction accuracy?


## What can we .lilac[do about it?]


???
So the questions that motivated this work were:
---
class: middle, left


## How do these different factors affect prediction accuracy?

#### .pink[Approach:] examine how well we can predict a highly heritable trait in different populations, based on an European ancestry discovery cohort and explore the roles of different biological/statistical factors

## What can we do about it?

#### .pink[Approach:] explore whether incorporating fine-scale ancestry information into these scores improves their performance

???
So the questions that motivated this work were:
---
class: middle, center

# .lilac[Let's look at] height
???
We chose to look at height in individuals with Eur, afr and admixed ancestry.
---
class:left, top
## Wow well can we predict a .lilac[highly heritable] trait?

&lt;img src="meat/schemas/schemas.017.png" width="1000px" /&gt;


???
why height? It's a somehwat well-udnerstood trait. it's hgihly popygenic and easy to measure. It is highly heritable (80% based ont twin studies, about 50% based on SNP heritability)

So in this work we used GWAS summary statistics for height from a cohort of 360K europeans from the UK biobank. And we verified how well we could predicti height in individuals with different degrees of african ancestry.

Importantly, unlike most studies, we consdierted ancestry as a continuous variable. So each individuals we analysed had a "proportion of Eur ancestry" inferred by local ancestry inference and averaged across the genome.

In the end we looked at about 19,000 individuals with admied ancestry, and 10,000 with eur ancestry from different cohorts.

we modell height as a function of sex age and eur ancesty and compare that to a mdoel that also includes prs.
---
class:left, top
## Wow well can we predict a .lilac[highly heritable] trait in .lilac[different populations?]
&lt;img src="meat/schemas/schemas.018.png" width="1000px" /&gt;

---
class:left, middle


## 1. .lilac[Introduction]

## 2. .lilac[Research Theme 1: Balancing selection in humans]

## 3. .lilac[Research Theme 2: Polygenic risk prediction for individuals with non-European ancestry] 

## 4. .pink[Conclusions &amp; Future Directions]
---
class: left, middle


### .pink[How has balancing selection shaped diversity in the HLA loci?]



### .pink[How prevalent has balancing selection been throughout evolution and what are the common trends?]


### .pink[How useful are polygenic risk predictors for individuals with non-European ancestries?]


---
class: middle, left



### .pink[1)] There is support for the .blue[divergent allele advantage model] in HLA evolution


### .pink[2)] A novel test, specifc for balancing selection, revealed many new candidates, 1/3 of which are immune-related, and many others related to defense broadly and reproduction


### .pink[3)] At present, PRS approaches have little to no utility for individuals with non-European ancestry, many factors are responsible, need to diversify genetics studies at all stages

---
class:left, top
&lt;img src="meat/schemas/schemas.012.png" width="1000px" /&gt;
---
class: left, top
## Research Axis I: Balancing selection as a force that shapes diversity

&lt;img src="meat/schemas/schemas.014.png" width="1000px" /&gt;
---
class: left, top
## Research Axis II: Polygenic risk prediction in admixed populations

&lt;img src="meat/schemas/schemas.013.png" width="930px" /&gt;
---
## A final thought

&lt;img src="meat/schemas/schemas.016.png" width="1000px" /&gt;
---
class: top, center
# Acknowledgements

.pull-left[.full-width[

**Iain Mathieson**

**Diogo Meyer**

**Aida Andrés**

Joshua Schmidt

Cesare de Filippo

Rodrigo dos Santos Francisco

Débora Brandt

]]

.pull-right[.full-width[
Neale Lab

UK Biobank

Women's Health Initiative

Jackson Heart Study

Health and Retirement Study

1000 Genomes Project
]]


&lt;img src="fapesp-logo.jpg" width="300px" /&gt;


---

class: inverse,center, middle

# .lilac[Thank you!]

# .lilac[Questions?]




    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"ratio": "16:9",
"slideNumberFormat": "<div class=\"progress-bar-container\">\n  <div class=\"progress-bar\" style=\"width: calc(%current% / %total% * 100%);\">\n  </div>\n</div>`\n"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
